COPYING.txt

```diff
@@ -4,4 +4,4 @@ http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
 Please see LICENSE.txt for the full text of the OSL 3.0 license or contact license@magentocommerce.com for a copy.
 
 Subject to Licensee's payment of fees and compliance with the terms and conditions of the MEE License, the MEE License supersedes the OSL 3.0 license for each source file.
-Please see <insert file name of the MEE license> for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
\ No newline at end of file
+Please see LICENSE_EE.txt for the full text of the MEE License or visit http://magento.com/legal/terms/enterprise.
```

Comment:
I don't think referring to a file which is actually missing in CE repository is correct.

Is there any problem with adding this file to CE repository also? It does not seem to me to bring any confusion as each module [defines](https://github.com/magento/magento2/blob/develop/app/code/Magento/Backend/composer.json#L26) applicable licenses.